### DISCUSSION TITLE
sgtelib confusion

### DISCUSSION THREAD

**Discussion by jrwrigh** [bbopt/NOMAD.jl#76] [Q&A]
_This is probably a NOMAD question more than anything else. Let me know if I should move it._

There's an option `sgtelib_model_search::Bool` that enables the use of the [SGTELIB](https://nomad-4-user-guide.readthedocs.io/en/latest/SgteLib.html) surrogate modeling library. This brings about 2 questions:

1. Which SGTELIB options are used? The [library documentation](https://nomad-4-user-guide.readthedocs.io/en/latest/SgteLib.html) lists many, many different settings to use, but the Julia library only given an on/off option.
2. If not the SGTELIB library, what other model is used for the search? This probably reveals my unfamiliarity with NOMAD (and optimization in general), but my understanding is that some form of surrogate model is being used to determine the black box values at mesh points, right? Or are all mesh points only the result of black-box evaluation?

**Comment by amontoison** [comment]
@salomonl Could you answer this question? I need your input as a core developer of NOMAD.
